packages/input/src/index.ts

```diff
@@ -78,6 +80,14 @@ export default createPrompt<string, InputConfig>((config, done) => {
     }
   });
 
+  // If prefill is set to 'editable' cut out the default value and paste into current state and the user's cli buffer
+  // They can edit the value immediately instead of needing to press 'tab'
+  useEffect((rl) => {
+    if (prefill !== 'editable' || !defaultValue) return;
+    rl.write(defaultValue);
+    setValue(defaultValue);
```

Comment:
```suggestion
    if (prefill === 'editable' && defaultValue) {
      rl.write(defaultValue);
      setValue(defaultValue);
    }
```

Reply:
Good callout, thanks for the code suggestion too, means I can rectify this on public transport :D